lib/players/human.rb

```diff
@@ -0,0 +1,14 @@
+module Players
+  # Inherits from Player.
+  class Human < Player
+
+    # Asks the user for input and returns it.
+    def move(board)
+      print "Where would you like to move (1-9)? "
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  puts "Where would you like to move?"
  gets.chomp
end
```